esx_service/utils/threadutils.py

```diff
@@ -84,13 +84,25 @@ def protected(*args, **kwargs):
     return lock_decorator
 
 
-def start_new_thread(target, args):
+def start_new_thread(target, args=None, daemon=False):
     """Start a new thread"""
-    threading.Thread(target=target, args=args).start()
+
+    new_thread = None
+
+    if args:
+        new_thread = threading.Thread(target=target, args=args)
```

Comment:
nit: is't L92-95 is the same as simply L93 ? Or threading package has different default for args ?